pom.xml

```diff
@@ -73,22 +74,22 @@
 	</distributionManagement>
 
 	<properties>
-		<!--
-			Maximum number of checkstyle violations allowed. Build will fail
-			if there are more than this number of violations. Feel free to
-			increase this number if you have a reason. The goal is not to be
-			dogmatic, but to provide consistency in the code.
-		-->
+		<!-- Maximum number of checkstyle violations allowed. Build will fail if 
```

Comment:
Seems your editor is overly eager to reformat existing code, and does not respect alignment (this is minor, just annoying when reading the PR, but I can definitely live with it).